package/MDAnalysis/analysis/align.py

```diff
@@ -229,21 +233,63 @@ def rotation_matrix(a, b, weights=None):
 
     See Also
     --------
-    :func:`rmsd` calculates the RMSD between *a* and *b*; for fitting a whole
+    :func:`rmsd` calculates the RMSD between *a* and *b*; for fitting a wholecan you also test that all warnings/exceptions are raised. The best thing is to make sure with coverage that the whole class is covered.
+
     trajectory it is more efficient to use :func:`rms_fit_trj`. A complete fit
     of two structures can be done with :func:`alignto`. """
-    if weights is not None:
-        # weights are constructed as relative to the mean
-        weights = np.asarray(weights, dtype=np.float64) / np.mean(weights)
 
     a = np.asarray(a, dtype=np.float64)
     b = np.asarray(b, dtype=np.float64)
     N = b.shape[0]
 
+    if weights is not None:
+        #qcp does NOT divide weights relative to the mean
+        weights = np.asarray(weights, dtype=np.float64) / np.mean(weights)
+
+
     rot = np.zeros(9, dtype=np.float64)
+    # Need to transpose coordinates such that the coordinate array is
+    # 3xN instead of Nx3. Also qcp requires that the dtype be float64
+    # (I think we swapped the position of ref and traj in CalcRMSDRotationalMatrix
+    # so that R acts **to the left** and can be broadcasted; we're saving
+    # one transpose. [orbeckst])
     rmsd = qcp.CalcRMSDRotationalMatrix(a.T, b.T, N, rot, weights)
     return np.matrix(rot.reshape(3, 3)), rmsd
 
+def _fit_to(mobile_coordinates, ref_coordinates, mobile_atoms,\
```

Comment:
**PEP8** two lines between function definitions. The best is to run a formatter like yapf or autopep8. The result should mostly be ok but check the diff.